mapbox-gl/mapbox-gl-tests.ts

```diff
@@ -0,0 +1,445 @@
+// These examples are from Mapbox (https://www.mapbox.com/mapbox-gl-js/examples/) and therefore under the copyright of Mapbox Inc.
```

Comment:
I would prefer language like `// These examples adapted from Mapbox's examples (link)` without any possible copyright confusion at the end. I don't want to make it look like we're saying "We copied this but it's still theirs" since it's either fair use (thus OK) or not (in which case we can't do it).